ansible/day_0/config-vmanage.yml

```diff
@@ -41,12 +41,11 @@
       until: result is not failed 
 
     - name: Add the Enterprise CA
-      vmanage_settings:
-        host: "{{ vmanage_ip }}"
+      cisco.sastre.settings_enterprise_ca:
+        address: "{{ vmanage_ip }}"
         user: "{{ vmanage_user }}"
```

Comment:
We may want to switch to using DNS names for vManage eventually. Is this restricted to IP addresses right now?